Tools/scripts/pathfix.py

```diff
@@ -164,12 +180,28 @@ def fix(filename):
     # Return success
     return 0
 
+
+def parse_shebang(shebangline):
+    end = len(shebangline)
+    start = shebangline.find(b' -', 0, end)  # .find() returns index at space
```

Comment:
Please remove the comment, IMHO it's not really helpful.

", 0, end" are the default value and so can be removed.